package.json

```diff
@@ -68,7 +72,7 @@
         "build-www": "cd www && yarn build",
         "start-www": "cd www && yarn start",
         "start-tests": "webpack-dev-server -d --config webpack.tests.config.js",
-        "test-browser": "ts-node -- ./scripts/with-http-server.ts -C dist/test -p 8079 -- mocha-webdriver-runner http://localhost:8079/index.html --chrome",
+        "test-browser": "yarn karma-headless",
```

Comment:
the test-browser script is for keeping the CI working, but we should remove it in a follow up change.